js/control/TrackAnalysis.js

```diff
@@ -175,14 +176,19 @@ BR.TrackAnalysis = L.Class.extend({
                                 segments[segmentIndex].feature.properties.messages[messageIndex][3]
                             );
                             break;
+                        case 'maxspeed':
                         case 'surface':
                         case 'smoothness':
                             if (typeof analysis[tagName][wayTagParts[1]] === 'undefined') {
+                                let formattedName = i18next.t([
+                                    'sidebar.analysis.data.' + tagName + '.' + wayTagParts[1],
+                                    wayTagParts[1],
+                                ]);
+                                if (tagName.indexOf('maxspeed') === 0) {
+                                    formattedName += ' km/h';
```

Comment:
nit: not possible to have it within the i18n content directly? 

Eg add `sidebar.analysis.data.maxspeed.XXX` and in English set to `%s km/h`?

Reply:
Yes, good catch. I'll change it in a follow-up PR.